content/post/2017-05-30-databases-and-sql.Rmd

```diff
@@ -151,9 +151,61 @@ It can also be easily read by any software or parsed at the command line using t
 > where are redundancies?
 
 
-## Lets look at the underlying database
+* Site + city + lat + lon
+* Date + time + timezone
+* genus + scientificname + common name 
+* Cultivar
+* Trait + units & description 
+* Mean 
+* Method
+* Entity
+
+### Tidy data
+
+
+* Each variable forms a column.
+* Each observation forms a row.
+* Each type of observational unit forms a table.
+
+From [tidyr vignette](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html)
+
+> YOUR TURN Is the Miscanthus dataset tidy? What would we need to do to make it tidy?
+
+## Relational Databases and SQL
```

Comment:
This section was left empty, should we link to some primer on relational databases and SQL?